client/src/pages/reports/Show.js

```diff
@@ -673,29 +673,31 @@ const ReportShow = ({ setSearchQuery, pageDispatchers }) => {
                   }
                 />
 
-                <Field
-                  name="attachments"
-                  label={Settings.fields.attachment.shortLabel}
-                  component={FieldHelper.ReadonlyField}
-                  humanValue={
-                    <div
-                      id="attachmentCardList"
-                      style={{
-                        display:
-                          report.attachments.length > 1 ? "flex" : "block"
-                      }}
-                    >
-                      {report.attachments.map((attachment, index) => (
-                        <AttachmentCard
-                          key={index}
-                          attachment={attachment}
-                          index={index}
-                          edit={false}
-                        />
-                      ))}
-                    </div>
-                  }
-                />
+                {Settings.fields.attachment.featureDisabled && (
```

Comment:
Shouldn't this be `!Settings.fields.attachment.featureDisabled && (…` i.e. the inverse condition?

Reply:
https://github.com/NCI-Agency/anet/tree/AB-894-permission-control-to-upload-attachments I did it, github status have error so its not display here